### PR TITLE
Reset password by send link via email

### PR DESCRIPTION
**Pull request by dangvm-0369**
## Related Tickets:
- [#69360](https://edu-redmine.sun-asterisk.vn/issues/69360)

## WHAT this PR does?
- Send mail reset password with BE reset password Breeze
- Customize mail send to user

## Notes Impacted Areas:
- N/A.

## Attachments:
Screenshot 2023-09-20 000328

### CODE REVIEW

**Comment by dangvm-0369** [comment]
ready

**Comment by dangvm-0369** [comment]
ready

File: app/Notifications/ResetPasswordNotification.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        $url = route('password.reset', ['token' => $this->token, 'email' => $notifiable->email]);
+
+        return (new MailMessage)
+            ->subject(__('Reset Password Notification'))
+            ->greeting(__('Hello') . ' ' . $notifiable->full_name . '!')
```
Comment (ghost) [inline app/Notifications/ResetPasswordNotification.php]: 'Hello :full_name !' :x ko nối chuỗi ntn

**Comment by dangvm-0369** [comment]
ready

File: app/Notifications/ResetPasswordNotification.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        $url = route('password.reset', ['token' => $this->token, 'email' => $notifiable->email]);
+
+        return (new MailMessage)
+            ->subject(__('Reset Password Notification'))
+            ->greeting(__('Hello :full_name !', ['full_name' => $notifiable->full_name]))
+            ->line(__('You are receiving this email because we received a password reset request for your account.'))
+            ->action(__('Reset Password'), $url)
+            ->line(__('This password reset link will expire in :count minutes.', [
+                'count' => config('auth.passwords.users.expire')
+                ]))
```
Comment (ghost) [inline app/Notifications/ResetPasswordNotification.php]:
```suggestion
            ->line(__('This password reset link will expire in :count minutes.', [
                'count' => config('auth.passwords.users.expire'),
            ]))
```

**Comment by dangvm-0369** [comment]
ready

**Pull request merged**